src/link_prober/IcmpPayload.h

```diff
@@ -42,19 +42,76 @@ __BEGIN_DECLS
 
 namespace link_prober
 {
+/**
+ *@enum TlvType
+ *
+ *@brief Supported TLV types
+ */
+enum TlvType: uint8_t {
+    TLV_COMMAND = 1,
+    TLV_SENTINEL = 0x0f,
```

Comment:
I suggest assigning `TLV_SENTINEL` to `0` since it's used as flag. And the type value larger than 0 is considered as common types.

Reply:
Uninitialized buffer has zeros inside, so I defined sentinel type to be `0xff`.